agent/lib/kontena/launchers/etcd.rb

```diff
@@ -85,38 +85,47 @@ def create_data_container(image)
     # @param [String] image
     # @param [Hash] info
     def create_container(image, info)
```

Comment:
The return value in this case is actually not used, but for consistency sake it makes sense to always return the container.